ports/unix/mpconfigport.h

```diff
@@ -325,3 +325,4 @@ void mp_unix_mark_exec(void);
 
 #define MICROPY_PY_BUILTINS_HELP (1)
 #define MICROPY_PY_BUILTINS_HELP_MODULES (1)
+#define CIRCUITPY_RE_DEBUG (1)
```

Comment:
Why don't we make this 0 by default? I don't think the `re1_5_dumpcode()` serves any purpose for the normal user.

Reply:
OK, split out into #4495